First working version with code extracted from Node.js 15 (#1)
uuidjs/randomUUID: README.md

# randomUUID

> Stub project for polyfilling `randomUUID` as proposed for standardization in

Reviewer: ```suggestion
Stub project for polyfilling `randomUUID` as proposed for standardization in
```

> https://github.com/WICG/uuid and implemented in
> [Node.js v15.6.0](https://github.com/nodejs/node/pull/36729).

Reviewer: ```suggestion
Polyfill for the `crypto.randomUUID()` method as proposed in
the [WICG randomUUID specification](https://github.com/WICG/uuid) and recently implemented in
[Node.js v15.6.0](https://github.com/nodejs/node/pull/36729).
```

## Usage

> Just `require()` this package as soon as possible in your project, so its
> functionality is available for other modules:

Reviewer: ```suggestion
Require module to polyfill `crypto.randomUUID()` polyfill (if needed).
```

```js
require('randomuuid')
```

> it will automatically detect and apply the polyfill functions in place, both in
> Node.js and browser code.

Reviewer: ```suggestion
```

> TODO: generate ESM version, so it can be directly importable by browsers, or
> better than that, confirm it's possible to write the polyfill directly on it.
> 

Reviewer: If ESM version won't be available soon, create an issue for it and link to that issue from here.
```suggestion
NOTE: ESM `import`able version not yet available.  See [this issue](appropriate link) for status.
```

## API

See Node.js documentation for
[randomUUID()](https://nodejs.org/dist/latest-v15.x/docs/api/crypto.html#crypto_crypto_randomuuid_options)
for API usage.

> Code is based on Node.js' `lib/internal/crypto/random.js`, so browser
> implementation (and by extension, Node.js `crypto.webcrypto` too) has some
> Node.js extra functionality that could not be available in future versions of
> the spec, more specifically the `disableEntropyCache` flag. This project intents
> to be keep up to date to the spec once it defines a browser API on itself.

Reviewer: Grammer nits.
```suggestion
Note: The current implementation is based on Node.js `lib/internal/crypto/random.js`, however the intent of this project is to follow the spec, so node-specific details (such as the `disableEntropyCache` option) may or may not survive in future versions.
```
